src/component.rs

```diff
@@ -81,25 +79,25 @@ impl Deref for TrailerSeparator {
         match self {
             TrailerSeparator::ColonSpace => ": ",
             TrailerSeparator::SpacePound => " #",
+            TrailerSeparator::__NonExhaustive => "",
```

Comment:
I changed this to `unimplemented` in my branch, I agree that the other way around (str to enum) should not panic, but using `__NonExhaustive` is undocumented and should not be supported in my opinion.

Reply:
Good idea.  I think `unreachable` might be more correct.

Reply:
Yeah, good call.